src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/IProjectHotReloadSessionCallback.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.HotReload
+{
+    public interface IProjectHotReloadSessionCallback
+    {
+        bool IsSupported { get; }
```

Comment:
@BillHiebert I don't think we need this. Presumably the code implementing the callback is also the code calling `IProjectHotReloadAgent.CreateHotReloadSession`--so if it knows Hot Reload isn't supported it just wouldn't create a session to being with.

Reply:
@tmeschter.  At first I was unsure where all these properties would live so was unsure which code would know whether it is supported or not.  But I agree, if the project already has a setting to disable it, it should read that and not call to start a hot reload session